src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -354,9 +354,11 @@ public function toggleStatusAction(int $productId): JsonResponse
         $isEnabled = $this->getQueryBus()->handle(new GetProductIsEnabled((int) $productId));
 
         try {
-            $this->getCommandBus()->handle(
-                new UpdateProductStatusCommand((int) $productId, !$isEnabled)
-            );
+            $this->getCommandBus()->handle(new UpdateProductStatusCommand(
```

Comment:
Same suggestion as above, the `GetProductIsEnabled` should also be multishop if you want to get the correct initial value to revert